.pre-commit-config.yaml

```diff
@@ -39,10 +44,7 @@ repos:
     rev: 3.9.2
     hooks:
     -   id: flake8
-        additional_dependencies:
-            - flake8-comprehensions==3.1.0
-            - flake8-bugbear==21.3.2
-            - pandas-dev-flaker==0.2.0
+        additional_dependencies: *version_check_dependencies
```

Comment:
what about the `yesqa` hook?

Reply:
sorry sir , i just made a commit resolving yesqa hook